src/RecoveryServices/RecoveryServices.SiteRecovery/RecoveryServices.SiteRecovery.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup>
     <AssemblyName>$(AzAssemblyPrefix)$(PsModuleName).SiteRecovery</AssemblyName>
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName).SiteRecovery</RootNamespace>
+    <WarningsNotAsErrors>612,618</WarningsNotAsErrors>
```

Comment:
Do not add this directive.  If you need to suppress warnings, please do an in source suppression at the particular location, rather than a blanket suppression for the warning type, like this.

Reply:
This is not required now. When we were using obsolete parameter dotnet compile was failing .To over ride that these tag was added. Not needed anymore.